quiver_engine/server.py

```diff
@@ -133,7 +138,11 @@ def run_app(app, port=5000):
 
 def launch(model, classes=None, top=5, temp_folder='./tmp', input_folder='./',
            port=5000, html_base_dir=None, mean=None, std=None):
-    os.system('mkdir -p %s' % temp_folder)
+    if platform.system() is 'Windows':
+        temp_folder = '.\\tmp'
```

Comment:
style nitpick: reassigning value to an input argument to the function is an ugly practice. I can merge it for now and I'll fix it when refactoring later.